lib/libutils/ext/include/util.h

```diff
@@ -89,10 +89,9 @@
 /* x has to be of an unsigned type */
 #define IS_POWER_OF_TWO(x) (((x) != 0) && (((x) & (~(x) + 1)) == (x)))
 
-#define ALIGNMENT_IS_OK(p, type) \
-	(((uintptr_t)(p) & (__alignof__(type) - 1)) == 0)
-
-#define IS_ALIGNED(x,a) (!((x) & ((a) - 1)))
+#define IS_ALIGNED(x,a)			(!((x) & ((a) - 1)))
```

Comment:
add a space char after the comma.

Another comment: I know optee guys prefer `!x` over `x == 0` but here, seen all the parentheses, i think `(((x) & ((a) - 1)) == 0)` is more readable to anyone.